pkg/scheduler/api/resource_info.go

```diff
@@ -187,10 +187,14 @@ func (r *Resource) Add(rr *Resource) *Resource {
 	return r
 }
 
-//Sub subtracts two Resource objects.
+// Sub subtracts two Resource objects with assertion.
```

Comment:
Although this modification is clear, I think the original function is also acceptable.